components/ContactForm.js

```diff
@@ -25,7 +35,5 @@ function ContactForm() {
     </form>
   );
 }
-function App() {
-  return <ContactForm />;
-}
-export default App;
```

Comment:
This extra `App` wrapper seemed unintentional but let me know if it's doing something!